XCUITests/DataManagementTest.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DataManagementTest.swift
+//  XCUITests
+//
+//  Created by Meera Rachamallu on 9/4/18.
+//  Copyright © 2018 Mozilla. All rights reserved.
+//
+
+import XCTest
+
+class DataManagementTest: BaseTestCase {
+    func testCheckDataManagementSettingsByDefault() {
+        navigator.goto(WebsiteDataSettings)
```

Comment:
After this line it would be good to check that the first time we go to the menu the table is empty, there are no entries. You can use something like:

`XCTAssertEqual(app.tables.cells.count, 0) `

See example [here](https://github.com/mozilla-mobile/firefox-ios/blob/a8f5f433e127ccf4b4378bbd77036815c50aa31a/XCUITests/TabTraySearchTabsTests.swift#L22)

Reply:
Also, I would add some checks to assure that the options shown are correct. For example, that there is the Show More option, the Edit button and so